packages/grafana-data/src/datetime/timezones.ts

```diff
@@ -22,6 +22,10 @@ export const timeZoneFormatUserFriendly = (timeZone: TimeZone | undefined) => {
   }
 };
 
+export const getZone = (timeZone: string) => {
+  return moment.tz.zone(timeZone);
```

Comment:
Does this one support the "grafana specific" time zones? `browser` and I think there should be another one...hmm..don't remember 🙈